### DISCUSSION TITLE
Lesson 7- unable to use make command in WSL

### DISCUSSION THREAD

**Discussion by andrejscek** [Cyfrin/foundry-full-course-cu#571] [Q&A] [answered]
Hello!

Is anyone able to use the make command running in WSL?
When I first tried to run the make command I got this response:
```
Command 'make' not found, but can be installed with:
sudo apt install make        # version 4.3-4.1build1, or
sudo apt install make-guile  # version 4.3-4.1build1
```

I then installed the make tool with the first command, but now it seems that the make tool is unaware of the MakeFile:
```
XXX:~/sol-course/les_7$ make build
make: *** No rule to make target 'build'.  Stop.
```

Any ideas appreciated, thanks

**Comment by shikhar229169** [comment, marked as answer]
Actually you need to declare all the commands in Makefile which you want to use with make.

You can have a look how Makefile looks like: https://github.com/Cyfrin/foundry-fund-me-f23/blob/main/Makefile
It is from the FundMe lesson.

**Reply by andrejscek** [reply]
Thank you, I just needed to read the code again, it turns out I wrote it is a "MakeFile" instead of "Makefile" and that is why it was not picking it up
